test/e2e/flask/solana/common-solana.ts

```diff
@@ -59,7 +59,7 @@ export const commonSolanaTxConfirmedDetailsFixture = {
   amount: '0.00708 SOL',
   networkFee: '0.000005 SOL',
   fromAddress: 'HH9ZzgQvSVmznKcRfwHuEphuxk7zU5f92CkXFDQfVJcq',
-  toAddress: 'AL9Z5JgZdeCKnaYg6jduy9PQGzo3moo7vZYVSTJwnSEq',
+  toAddress: '4tE76eixEgyJDrdykdWJR1XBkzUk4cLMvqjR2xVJUxer',
```

Comment:
is this supposed to be part of the PR?

Reply:
yes, it's a fix to support receive transactions on the test